editor/src/components/inspector/constraints-section.tsx

```diff
@@ -51,10 +52,19 @@ export const ConstraintsSection = React.memo(() => {
     'ConstraintsSection onlyGroupChildrenSelected',
   )
 
+  const showSection = React.useMemo(() => {
+    return noGroupOrGroupChildrenSelected || onlyGroupChildrenSelected
+  }, [noGroupOrGroupChildrenSelected, onlyGroupChildrenSelected])
```

Comment:
With this we should also remove the `when`s below

Reply:
the two `when`s are not necessarily true at the same time though, I think they should still remain (one can be true while the other is false)

Reply:
Sorry, my bad. Disregard that!